dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresVectorStoreOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.Postgres;
+
+/// <summary>
+/// Options when creating a <see cref="PostgresVectorStore"/>.
+/// </summary>
+public sealed class PostgresVectorStoreOptions
+{
+    /// <summary>
+    /// Gets or sets the default vector size to use when creating a new vector.
+    /// </summary>
+    public int DefaultVectorSize { get; init; } = 100;
```

Comment:
Is this for the dimensions of a vector because that is a required parameter for Vector Property attributes and on the definition, so a default shouldn't ever really be needed.